NevinChen.md

```diff
@@ -0,0 +1,11 @@
+## 10/26 - 10/28
+
+Android
+
+- Followed the document and prepared for development environment. [Taipei Mobile Team notes]
+
+- Bug 1310621 - Null Pointer exception while setting suggestion for search engines 
+- r+ & s.kaspari
```

Comment:
Actually, it shouldn't be `r?` but rather `WIP on the new approach given by the reviewer`.